### PR TITLE
Restructured authors (fixes #234)

### LINKED ISSUE #234: Restructure authorlist
I propose we restructure the authorlist on the first page.
Just state 'Authors' and then mention all the names (alphabetical order), with superscript 1-5 for all institutions.

### PR DESCRIPTION
**Pull request by redfast00**
#234

### CODE REVIEW

File: intro-HPC/title.tex [hunk 1]
```diff
@@ -25,24 +25,24 @@
...
+Stefan Becuwe\footnotemark[1],
+Geert-Jan Bex\footnotemark[2],
+Geert Borstlap\footnotemark[1],
+Bal\'azs Hajgat\'o\footnotemark[3],
+Kenneth Hoste\footnotemark[4],
+Kurt Lust\footnotemark[1],
+Mag Selwa\footnotemark[2],
+Bert Tijskens\footnotemark[1],
+Jens Timmerman\footnotemark[4],
+Toon Willems\footnotemark[4]
```
Comment (boegel) [inline intro-HPC/title.tex]: @redfast00 Don't forget to add yourself :)

**Comment by boegel** [comment]
@redfast00 Please also add:

* Stijn De Weirdt (UGent)
* Andy Georges (UGent)
* Samuel Moors (VUB)
* Ward Poelmans (VUB)
* Alvaro Simon Garcia (UGent)

(based on everyone who has committed to the repository, see https://github.com/hpcugent/vsc_user_docs/graphs/contributors)

@gjbex, @wpoely86, @hajgato, @klust, @backelj, @sbecuwe, @mselwa: Any objections on re-ordering authors here and dropping the distinction between main author & co-authors + sorting alphabetically?

**Comment by boegel** [comment]
@redfast00 Ah, one more...

* Álvaro Simón

@alvarosimon Isn't there a Garçia in there somewhere too? :)

**Comment by wpoely86** [comment]
:+1:

File: intro-HPC/title.tex [hunk 2]
```diff
@@ -25,24 +25,29 @@
...
-Bal\'azs Hajgat\'o (VUB)
-
+\normalsize\strong{Authors:}
+
+Franky Backeljauw\footnotemark[1],
+Stefan Becuwe\footnotemark[1],
+Geert-Jan Bex\footnotemark[2],
+Geert Borstlap\footnotemark[1],
+Jasper Devreker\footnotemark[4],
+Stijn De Wierdt\footnotemark[4],
```
Comment (boegel) [inline intro-HPC/title.tex]: `Weirdt` ;)

**Comment by boegel** [comment]
@redfast00 I'll hold off merging this until we got an OK from all sites that they're fine with changing this. Current status:

* [x] VUB (@wpoely86)
* [ ] KUL (@mselwa or @molden)
* [ ] UHasselt (@gjbex)
* [ ] UAntwerpen (@klust or @backelj)
* [x] UGent (@boegel)

**Comment by sbecuwe** [comment]
Ok for UAntwerpen.

**Comment by boegel** [comment]
Still awaiting feedback from KUL (@mselwa, @molden) and UHasselt (@gjbex)...

**Comment by mselwa** [comment]
Sorry, missed it. Ok for me.

**Comment by gjbex** [comment]
Two small points:
  * My name is officially Geert Jan, not Geert-jan, and
  * as affiliation, I'd prefer Hasselt University.

Thanks :)

**Comment by boegel** [comment]
Thanks for the feedback @gjbex, seems like your name was "misspelled" for a long time already, we'll get that fixed...

File: intro-HPC/title.tex [hunk 3]
```diff
@@ -25,24 +25,31 @@
...
+\'Alvaro Sim\'on Gar\c{c}\'ia\footnotemark[4],
+Bert Tijskens\footnotemark[1],
+Jens Timmerman\footnotemark[4],
+Toon Willems\footnotemark[4]
+
+\footnotetext[1]{UAntwerpen}
+\footnotetext[2]{KU Leuven}
+\footnotetext[3]{VUB}
+\footnotetext[4]{UGent}
+\footnotetext[5]{UHasselt}
```
[the remark this answers is not in the document]
Comment (sbecuwe) [reply intro-HPC/title.tex]: It should be "University of Antwerp" ;-)

File: intro-HPC/title.tex [hunk 4]
```diff
@@ -25,23 +25,31 @@
...
+Jasper Devreker\footnotemark[2],
+Stijn De Weirdt\footnotemark[2],
+Andy Georges\footnotemark[2],
+Bal\'azs Hajgat\'o\footnotemark[1],
+Kenneth Hoste\footnotemark[2],
+Kurt Lust\footnotemark[5],
+Samuel Moors\footnotemark[1],
+Ward Poelmans\footnotemark[1],
+Mag Selwa\footnotemark[4],
+\'Alvaro Sim\'on Gar\c{c}\'ia\footnotemark[2],
```
Comment (boegel) [inline intro-HPC/title.tex]: @redfast00 It's `García`, not `Garçía` (my bad)

**Pull request merged**